src/connector.rs

```diff
@@ -5,10 +5,14 @@ use fluvio_future::net::{
 use fluvio_ws_stream_wasm::WsMeta;
 use std::io::Error as IoError;
 #[derive(Clone, Default)]
-pub struct FluvioWebsocketConnector {}
+pub struct FluvioWebsocketConnector {
+    token: String
+}
 impl FluvioWebsocketConnector {
-    pub fn new() -> Self {
-        Self {}
+    pub fn new(token: String) -> Self {
```

Comment:
Maybe we can change this to

`protocols: Option<Vec<String>>` so it has the same shape than the WebSocket js just in case we need to add something new in the future, what do you think?

Reply:
Yeah, I thought about doing that, the `ws_stream_wasm` call takes `Option<Vec<&str>>` and so it'd be annoying.